addons/ai/CfgAmmo.hpp

```diff
@@ -0,0 +1,8 @@
+class CfgAmmo {
+	class RocketCore;
```

Comment:
Why define `RocketCore` when it is not used?

Reply:
Because rocketBase inherits from rocketCore?